File: risk-lib/src/test/java/net/mjduffin/risk/lib/entities/TerritoryTest.java
package net.mjduffin.risk.lib.entities;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class TerritoryTest {

    @Test
    void territoryInitialisationTest() {
        Player player = new Player("Bob");
        new Territory("Test", player);

        assertEquals(1, player.getTotalUnits());
    }



}

File: risk-lib/src/test/java/net/mjduffin/risk/lib/entities/TerritoryTest.kt
package net.mjduffin.risk.lib.entities

import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

internal class TerritoryTest {

    @Test
    fun territoryInitialisationTest() {

        var player = Player("Bob")
        var territory = Territory("Test")
        player.addTerritory(territory)

        assertEquals(1, player.totalUnits);

    }
}